subsys/bluetooth/mesh/Kconfig

```diff
@@ -0,0 +1,155 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if BT_MESH
+
+menu "nRF BT Mesh Models"
+
+config BT_MESH_NORDIC_MODELS
+	bool "nRF BT Mesh Model support"
+	help
+	  Common Mesh model support modules, required by all Nordic BT Mesh
+	  models.
+
+
+config BT_MESH_ONOFF_SRV
+	bool "Generic OnOff Server"
+	select BT_MESH_NORDIC_MODELS
+	help
+	  Enable Mesh Generic OnOff Server model.
+
+config BT_MESH_ONOFF_CLI
+	bool "Generic OnOff Client"
+	select BT_MESH_NORDIC_MODELS
+	help
+	  Enable Mesh Generic OnOff Client model.
+
+
+config BT_MESH_LVL_SRV
+	bool "Generic Level Server"
+	select BT_MESH_NORDIC_MODELS
+	help
+	  Enable Mesh Generic Level Server model.
+
+config BT_MESH_LVL_CLI
+	bool "Generic Level Client"
+	select BT_MESH_NORDIC_MODELS
+	help
+	  Enable Mesh Generic Level Client model.
+
+
+menuconfig BT_MESH_DTT_SRV
+	bool "Generic Default Transition Time Server"
+	select BT_MESH_NORDIC_MODELS
+	help
+	  Enable Mesh Generic Default Transition Time Server model.
+
+if BT_MESH_DTT_SRV
+
+config BT_MESH_DTT_SRV_PERSISTENT
```

Comment:
Is this config in use?

Does it have dependencies on SETTINGS or something like that?

Reply:
Looks like a sloppy bug